src/service/resource.js

```diff
@@ -493,12 +493,7 @@ export class Resource {
    * @return {!../layout-rect.LayoutRectDef}
    */
   getLayoutBox() {
-    if (!this.isFixed_) {
-      return this.layoutBox_;
-    }
-    const viewport = this.resources_.getViewport();
-    return moveLayoutRect(this.layoutBox_, viewport.getScrollLeft(),
-        viewport.getScrollTop());
+    return this.layoutBox_;
```

Comment:
Ok. I'll reverse this then and will add a new method for the non-offset box. From what I can tell, there's a near 50/50 split between them in downstream code.